.github/ISSUE_TEMPLATE/00-use-case.md

```diff
@@ -0,0 +1,25 @@
+---
+name: Use case
+about: Describe a new use case for the specification.
+title: ''
+labels: 'use case :bulb:'
+assignees: ''
+
+---
+
+**Description**
+
+As a:
+
+- [ ] Application user/user of the configuration itself
```

Comment:
What does "user of the configuration itself" mean and do we really expect users of applications that utilize MicroProfile Fault Tolerance to come to our issue tracker, or even know that MP FT is utilized in the application they use?

Reply:
We can reword to Application user, maybe.

Reply:
Since we have API user already, I have moved the Application user as I think they are the same.